.github/workflows/publish.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       - run: npm ci
       - run: npm run lint
       - run: npm run build --if-present
+      - run: npm run hardhat & npm run test:utils
       - run: npm run hardhat & npm run test:btc2x
       - run: npm run hardhat & npm run test:cdeti
       - run: npm run hardhat & npm run test:dseth
```

Comment:
do we have the option of just running `npm run hardhat & npm run test` instead of splitting it up across all of these tests?